kafka-ui-react-app/src/components/common/Input/Input.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useFormContext, RegisterOptions } from 'react-hook-form';
+
+import StyledIcon from './InputIcon.styled';
+import StyledInput from './Input.styled';
+
+export interface InputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {
+  name?: string;
+  hookFormOptions?: RegisterOptions;
+  inputSize?: 'M' | 'L';
+  leftIcon?: string;
+  rightIcon?: string;
+}
+
+const Input: React.FC<InputProps> = ({
+  className,
+  name,
+  hookFormOptions,
+  leftIcon,
+  rightIcon,
+  inputSize,
+  ...rest
+}) => {
+  const size = inputSize || 'L'; // default size is L
+  const methods = useFormContext();
+  return (
+    <div
+      style={{
+        position: 'relative',
+      }}
+    >
```

Comment:
let's move it to styles

Reply:
Yes, it may be named as StyledInputWrapper